src/app/(header-layout)/mypage/page.tsx

```diff
@@ -0,0 +1,56 @@
+'use client'
+
+import { signOut, useSession } from 'next-auth/react'
+import { useRouter } from 'next/navigation'
+import { useEffect } from 'react'
+import Image from 'next/image'
+export default function MyPage() {
+  const { data: session, status } = useSession()
+  const router = useRouter()
+
+  useEffect(() => {
+    if (status === 'unauthenticated') {
+      router.push('/login')
+    }
```

Comment:
if문 작성할때 한줄로 끝나는 경우에는 중괄호 사용하지 않는게 더 가독성이 좋을 것 같아요! 
```suggestion
    if (status === 'unauthenticated') router.push('/login')
```